waltz-ng/client/data-types/data-type-view.js

```diff
@@ -72,9 +78,17 @@ function controller($scope,
         .then(() => dataFlowService.loadDetail())
         .then(flowData => vm.flowData = flowData);
 
+
     dataTypeUsageStore
         .findForUsageKindByDataTypeIdSelector('ORIGINATOR', selector)
         .then(originators => vm.flowOriginators = originators);
+
+
+    dataTypeUsageStore
+        .findForUsageKindByDataTypeIdSelector('DISTRIBUTOR', selector)
+        .then(distributors => {
+            vm.flowDistributors = distributors
+        });
```

Comment:
It looks like I blindly hacked the first one in....  unless good reason not to, can we move both to `data-type-view-data` ?